nixos/tests/hardened.nix

```
@@ -24,14 +24,17 @@
       imports = [ ../modules/profiles/hardened.nix ];
       environment.memoryAllocator.provider = "graphene-hardened";
       nix.settings.sandbox = false;
-      virtualisation.emptyDiskImages = [ 4096 ];
-      boot.initrd.postDeviceCommands = ''
-        ${pkgs.dosfstools}/bin/mkfs.vfat -n EFISYS /dev/vdb
-      '';
+      virtualisation.emptyDiskImages = [
+        {
+          size = 4096;
+          driveConfig.deviceExtraOpts.serial = "EFISYS";
```

nit:
```suggestion
          driveConfig.deviceExtraOpts.serial = "efi";
```
or `"esp"`, since we’re not bound by FAT‐32 label format limitatinos now.

---

actually, it's not even an ESP at all. I'm just going to rename it to match the purpose of the test.